### ISSUE TITLE
A question in train_IMDN

### ISSUE THREAD

**Issue by fantasysponge** [Zheng222/IMDN#21]
the location of adjusting learning rate should be at the end of train function

**Comment by Zheng222** [comment]
@fantasysponge 
The previous pytorch (e.g., pytorch 1.1) did not have this warning. You can modify the location of adjusting learning rate by yourself.

**Comment by fantasysponge** [comment]
Thanks for your reply.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Zheng222/IMDN"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年10月11日(星期天) 晚上7:07
收件人:&nbsp;"Zheng222/IMDN"<IMDN@noreply.github.com&gt;;
抄送:&nbsp;"2044909893"<2044909893@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [Zheng222/IMDN] A question in train_IMDN (#21)





 
@fantasysponge
 The previous pytorch (e.g., pytorch 1.1) did not have this warning. You can modify the location of adjusting learning rate by yourself.
